src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Application.cshtml

```diff
@@ -149,7 +149,7 @@
                                 <span class="app-task-list__task-name app-task-list__task-name--inactive">
                                     @if (Model.ReadyToConfirm)
                                     {
-                                        <a href="/Roatp/Gateway/GoAndConfirm/@Model.ApplicationId">Confirm gateway outcome</a>
+                                        <a class="govuk-link" href="/Roatp/Gateway/@Model.ApplicationId/ConfirmOutcome">Confirm gateway outcome</a>
```

Comment:
Can tag helpers be used here instead of hard-coded urls

Reply:
Done - tag helpers used